lib/cdo/cloud_formation/cdo_app.rb

```diff
@@ -27,8 +27,8 @@ class CdoApp < StackTemplate
     # Hard-coded constants and default values.
     CHEF_BIN = '/usr/local/bin/chef-cdo-app'
     CHEF_KEY = rack_env?(:adhoc) ? 'adhoc/chef' : 'chef'
-    # Use AMI for Ubuntu 20 (ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20230517)
-    IMAGE_ID = ENV['IMAGE_ID'] || 'ami-0261755bbcb8c4a84'
+    # Use AMI for Ubuntu 20 (ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20240720)
+    IMAGE_ID = ENV['IMAGE_ID'] || 'ami-013b3de8a8fa9b39f'
```

Comment:
```suggestion
    # Use AMI for Ubuntu 22 for Adhocs, but keep using the Ubuntu 20 AMI for other environments.
    # All environments are using Ubuntu 22, but we are allowing some stack drift to avoid server replacement.
    IMAGE_ID = ENV['IMAGE_ID'] || 'ami-013b3de8a8fa9b39f'
```